pom.xml

```diff
@@ -126,6 +126,8 @@
                 <configuration>
                     <excludes>
                         <exclude>com/baidu/hugegraph/example/*.class</exclude>
+                        <exclude>com/baidu/hugegraph/util/IdUtil.class</exclude>
+                        <exclude>com/baidu/hugegraph/util/SplicingIdGenerator.class</exclude>
```

Comment:
also copy the unit test

Reply:
Only exist IdUtilTest.class in core

Reply:
add test for SplicingIdGenerator, also add to hugegraph-server